src/saveload/saveload.h

```diff
@@ -352,6 +352,7 @@ enum SaveLoadVersion : uint16 {
 
 	SLV_EXTEND_VEHICLE_RANDOM,              ///< 310  PR#10701 Extend vehicle random bits.
 	SLV_EXTEND_ENTITY_MAPPING,              ///< 311  PR#10672 Extend entity mapping range.
+	SLV_ECONOMY_DATE,                       ///< 312  PR#10700 Split calendar and economy timers and dates.
```

Comment:
Strictly seen we don't need this new version, as introducing a new saveload field "just works". But I guess we are so used to it .. so it is fine.